generate-geodatabase-replica-from-feature-service/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        tools:context=".MainActivity">
+
+        <com.arcgismaps.mapping.view.MapView
+            android:id="@+id/mapView"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent">
+
+        </com.arcgismaps.mapping.view.MapView>
+
+        <Button
```

Comment:
```suggestion
        <com.google.android.material.button.MaterialButton
```
Just a suggestion, but would be nice if the sample could use Google's `MaterialButton` and use the style `?attr/materialButtonOutlinedStyle`. You could refer to other samples that use the button.

Reply:
Thanks, that does look better.